[2025-06-18 15:07:23,273] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2024-06-06T00:00:00+00:00 [queued]>
[2025-06-18 15:07:23,315] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2024-06-06T00:00:00+00:00 [queued]>
[2025-06-18 15:07:23,315] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:07:23,315] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 15:07:23,315] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:07:23,430] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2024-06-06T00:00:00+00:00
[2025-06-18 15:07:23,456] {standard_task_runner.py:52} INFO - Started process 13435 to run task
[2025-06-18 15:07:23,466] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'second_task', '2024-06-06T00:00:00+00:00', '--job-id', '12787', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpo0o_k67x', '--error-file', '/tmp/tmpkif1o5yq']
[2025-06-18 15:07:23,475] {standard_task_runner.py:77} INFO - Job 12787: Subtask second_task
[2025-06-18 15:07:23,645] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.second_task 2024-06-06T00:00:00+00:00 [running]> on host 500dc4ac3a72
[2025-06-18 15:07:23,852] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=my_name
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2024-06-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-06T00:00:00+00:00
[2025-06-18 15:07:23,854] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-18 15:07:23,860] {bash.py:158} INFO - Running command: echo "this is sencond task "
[2025-06-18 15:07:23,880] {bash.py:169} INFO - Output:
[2025-06-18 15:07:23,892] {bash.py:173} INFO - this is sencond task
[2025-06-18 15:07:23,893] {bash.py:177} INFO - Command exited with return code 0
[2025-06-18 15:07:24,040] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, execution_date=20240606T000000, start_date=20250618T150723, end_date=20250618T150724
[2025-06-18 15:07:24,220] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 15:07:24,317] {local_task_job.py:146} INFO - Task exited with return code 0
